README.md

```diff
@@ -26,12 +26,12 @@ Yes nothing is free...
 * TODO: more Useful blog links
 
 ### Some tech stuffs
-Nothing private uses the browser fingerprinting feature of client.js to obtain the fingerprint of your web browser. It will save the fingerprint along with your name in a Sqlite db using PHP as backend, when you submits it. Next time, when you visit the website, your browser fingerprint is matched with the cloumn in the database and it will return your name.
+Nothing private uses the browser fingerprinting feature of Client.js to obtain the fingerprint of your web browser. It will save the fingerprint along with your name in a Sqlite db using PHP as backend, when you submits it. Next time, when you visit the website, your browser fingerprint is matched with the column in the database and it will return your name.
```

Comment:
`Sqlite` -> `SQLite`.
`db` -> `DB` or `database`.
`submits` -> `submit`
`Next time, when you visit the website` -> `Next time you visit the website`